docs/contributing/setting-up-your-local-dev-environment.md

```diff
@@ -76,6 +76,12 @@ git checkout package.json; yarn --force
   - To run tests for a single package you can run: `yarn jest <package-name>`.
   - To run a single test file you can run: `yarn jest <file-path>`.
 
+If you're adding e2e tests and want to run them against local changes:
+
+- In the root of the mono repo run `yarn lerna run build --scope=<package-name>` where `package-name` is the changes you're testing
```

Comment:
```suggestion
- In the root of the monorepo, run `yarn lerna run build --scope=<package-name>` where `package-name` is the directory containing the changes you're testing.
```

I found "is the changes you're testing" a little bit awkward. Feel free to fiddle further! 💖